bindings/python/iota_sdk/client/_utils.py

```diff
@@ -1,20 +1,43 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
+from typing import Optional
+from abc import ABCMeta, abstractmethod
+
 from iota_sdk.types.common import HexStr
 from iota_sdk.types.output import Output
-from typing import Optional
 
 
-class ClientUtils():
+class ClientUtils(metaclass=ABCMeta):
     """Client utility functions.
     """
 
-    def hex_to_bech32(self, hex: HexStr, bech32_hrp: str) -> str:
+    @abstractmethod
+    def _call_method(self, name, data=None):
+        """
+        Sends a message to the Rust library and returns the response.
+        It is abstract here as its implementation is located in `client.py`, which is a composite class.
```

Comment:
Shouldn't this be utils.py?

Reply:
No, this is implemented in client.py only.
`class Client(NodeCoreAPI, NodeIndexerAPI, HighLevelAPI, ClientUtils):`